addons/website/static/src/js/widgets/theme.js

```diff
@@ -509,7 +509,7 @@ var ThemeCustomizeDialog = Dialog.extend({
                 });
 
                 var colors = {};
-                colors[colorName] = ev.data.hex;
+                colors[colorName] = 'rgba('+ev.data.red+' , '+ev.data.green+' , '+ev.data.blue+' , '+ev.data.opacity+')';
```

Comment:
You should not have to do this. Replace `hex` in the color components by 'cssColor' which will either be an hex value if opacity == 1 or a rgba(...) string if there is an opacity != 1